_data/fr/settings.yml

```diff
@@ -21,12 +21,14 @@ banner:
     secure_description: 'Un <strong>verrou</strong> ou <strong>https://</strong> signifie que vous êtes connecté en toute sécurité au site Web .gov. Partagez des informations sensibles uniquement sur des sites Web officiels et sécurisés.'
 contact_page:
   operating_hours: Les heures d'ouverture sont du lundi au vendredi de 8 h à 20 h (heure de l'Est).
+  required: obligatoire
   call_us:
     title: Appelez-nous
   support_request_form:
     title: Soumettre un ticket d'assistance
     other: Other
     help_text: Veuillez prévoir 2 jours ouvrables pour recevoir une réponse. Nous vous remercions de votre patience.
+    required_text: Les champs obligatoires sont marqués d'un astérisque
     agencies:
       'Central Intelligence Agency': Agence centrale de renseignement
       'Consumer Financial Protection Bureau': Bureau financier de la protection des consommateurs
```

Comment:
In the Spanish and English versions, there is text that will serve as a link to visit USA.gov. I am not seeing a similar translation in French. Is this intentional?

Reply:
So I think the text is there, It just was some weird styling issue that it caught since its the last line on the page.

Reply:
so I dont think its breaking anything.